layers/+lang/php/README.org

```diff
@@ -18,15 +22,38 @@ This layer adds PHP language support to Spacemacs.
 - Complete and jump to define with [[https://github.com/xcwen/ac-php][company-php]]
 - Run tests with PHPUnit
 - Reformat code with PHP CBF
+- Support for the [[https://langserver.org/][Language Server Protocol]] (experimental)
 
 The =gtags= layer is recommended to benefit from better =eldoc= and
 =helm-gtags=.
 
 * Install
+** Layer
 To use this configuration layer, add it to your =~/.spacemacs=. You will need to
 add =php= to the existing =dotspacemacs-configuration-layers= list in this
 file.
 
+** Setting the backend
+To activate the backend set the layer variable =php-backend=:
+
+#+BEGIN_SRC elisp
+  (php :variables php-backend 'lsp)
+#+END_SRC
+
+For spacemacs to use lsp you also need to add =lsp= to the
+=dotspacemacs-configuration-layers= in your =~/.spacemacs= file.
+
+* Backend
+** Language Server Protocol
+You have to install the php language server:
+
+#+BEGIN_SRC sh
```

Comment:
Based on the users' feedback https://github.com/bmewburn/vscode-intelephense was recommended as the better server(and it is easier to install). IMO this should be mentioned here.

Reply:
Ok, thanks for the info. I'll play around with it once I find the time. Maybe we can use it instead of `php-language-server` then or let the user toggle between the two like in the c++ layer.

Reply:
> let the user toggle between the two like in the c++ layer.

IMO better to use lsp-mode facilities to switch the server. C++ case is a bit different since there are a lot of functions that are language specific and also they require additional Emacs package to work while the php servers are bundled in lsp-mode.

Reply:
I've now tested intelephense and I agree that it's the better server. I updated the README and the changelog.